gm4_crossbow_cables/data/crossbow_cables/functions/recipe_check.mcfunction

```diff
@@ -0,0 +1,13 @@
+#spooled crossbow
+
+#neither have been used
+execute if score @s gm4_slot_count matches 2 if score @s gm4_stack_size matches ..1 if block ~ ~ ~ dropper{Items:[{Slot:0b,id:"minecraft:crossbow"},{Slot:4b,id:"minecraft:fishing_rod"}]} unless data block ~ ~ ~ Items[0].tag.Damage unless data block ~ ~ ~ Items[4].tag.Damage run data merge block ~ ~ ~ {Items:[{Slot:8,id:"minecraft:crossbow",Count:1b,tag:{gm4_spooled_crossbow:1b,gm4_custom_crafters:{multiplier:1},display:{Lore:["{\"text\":\"Use with string in the offhand\",\"color\":\"dark_aqua\",\"italic\":\"false\"}","{\"text\":\"to create lines of tripwire.\",\"color\":\"dark_aqua\",\"italic\":\"false\"}"],Name:"{\"text\":\"Spooled Crossbow\",\"italic\":\"false\"}"}}}]}
```

Comment:
These recipe checks should check to make sure the crossbow being used in the recipe is not already a spooled crossbow. I can currently do this.